builds/misc/packages-release.yaml

```diff
@@ -473,22 +440,6 @@ stages:
           identityServiceArtifactName: packages_redhat-ubi9-latest_amd64
           identityServicePackageFilter: aziot-identity-service-*.x86_64.rpm
 
-        Debian10-amd64:
-          os: debian10
-          artifactName: iotedged-debian10-amd64
-          identityServiceArtifactName: packages_debian-10-slim_amd64
-          identityServicePackageFilter: aziot-identity-service*_amd64.deb
-        Debian10-arm32v7:
-          os: debian10
-          artifactName: iotedged-debian10-arm32v7
-          identityServiceArtifactName: packages_debian-10-slim_arm32v7
-          identityServicePackageFilter: aziot-identity-service*_armhf.deb
-        Debian10-aarch64:
-          os: debian10
-          artifactName: iotedged-debian10-aarch64
-          identityServiceArtifactName: packages_debian-10-slim_aarch64
-          identityServicePackageFilter: aziot-identity-service*_arm64.deb
-
```

Comment:
Does the Cx facing doc team know about these OS drop?

Reply:
Yes, we're working with them.